solcx/install.py

```diff
@@ -102,14 +102,13 @@ def get_solcx_install_folder(solcx_binary_path: Union[Path, str] = None) -> Path
         return path
 
 
-def _get_which_solc() -> Path:
+def _get_which_solc() -> Optional[Path]:
```

Comment:
i think there is 1 more place at the module level to handle the optional.
i am happy to take of this! we are dusting this thing off